.github/workflows/main.yml

```diff
@@ -39,13 +39,17 @@ jobs:
           submodules: recursive
 
       - name: Go build
-        run: CGO_ENABLED=0 GOOS=linux go build -a -o artifacts/polygon-sdk .
+        run: CGO_ENABLED=1 GOOS=linux go build -race -a -o artifacts/polygon-sdk .
 
+      - name: Go build with race 
+        run: CGO_ENABLED=1 GOOS=linux go build -race -a -o artifacts/polygon-sdk-race .
+      
       - name: Add artifacts directory to the path
         run: echo "$(pwd)/artifacts" >> $GITHUB_PATH
 
       - name: Go test
-        run: go test -timeout 20m ./...
+        run: go test -timeout 28m ./...
+        run: ls
```

Comment:
What happens when you put two runs like this? Why `1s`?

Reply:
It was mistake, Removed it. Commit [link](https://github.com/0xPolygon/polygon-sdk/pull/207/commits/4ae18f2be64714f933d5e5b421737c831b2e9e6a)